File: api/src/main/kotlin/org/team619/rutor/model/BlankElements.java
package org.team619.rutor.model;

import org.jsoup.select.Elements;

/**
 * Created by BORIS on 27.08.2016.
 */
public class BlankElements extends Elements {

    public static Elements newList() {
        return new BlankElements();
    }

    @Override
    public String toString() {
        return "";
    }

}

File: api/src/main/kotlin/org/team619/rutor/model/BlankElements.kt
package org.team619.rutor.model

import org.jsoup.select.Elements

/**
 * Created by BORIS on 27.08.2016.
 */
class BlankElements : Elements() {

    override fun toString(): String {
        return ""
    }

    companion object {

        fun newList(): Elements {
            return BlankElements()
        }
    }

}
